WinImageBuilder.psm1

```diff
@@ -792,6 +829,53 @@ function Run-Sysprep {
 }
 
 function New-MaaSImage {
+    <#
+    .SYNOPSIS
+     This function generates a MaaS ready image from an ISO file.
+    .DESCRIPTION
+     It calls New-WindowsOnlineImage with the Type parameter set to MAAS.
+    .PARAMETER WimFilePath
+     The location of the WIM file from the Windows ISO.
+    .PARAMETER ImageName
+     This is the object which contains all the information about the Windows flavor selected.
+    .PARAMETER MaaSImagePath
+     The destination of the generated image.
+    .PARAMETER SizeBytes
+     The size of the VHD used to generate the image.
+    .PARAMETER DiskLayout
+     This parameter can be set to either BIOS or UEFI.
+    .PARAMETER ProductKey
+     The product key for the OS selected.
+    .PARAMETER VirtIOISOPath
+     The path to the ISO file containing the VirtIO drivers.
+    .PARAMETER InstallUpdates
+     If set to true, the latest updates will be downloaded and installed.
+    .PARAMETER AdministratorPassword
+     Is used by the script to connect to the instance while it is generating.
+    .PARAMETER PersistDriverInstall
+     In case the hardware on which the image is generated will also be the hardware on
+     which the image will be deployed this can be set to true.
+    .PARAMETER ExtraFeatures
+     Name of the extra features to enable on the generated image.
+    .PARAMETER ExtraDriversPath
+     The path to the additional drivers to install.
+    .PARAMETER Memory
+     RAM assigned to the VM used to generate the image.
+    .PARAMETER CpuCores
+     The number of CPU cores assigned to the VM used to generate the image.
+    .PARAMETER RunSysprep
+     Used to clean the OS on the VM, and to prepare it for a first-time use.
+    .PARAMETER SwitchName
+     Used to specify the virtual switch the VM will be using to connect to the internet.
+     If none is specified, one will be created.
+    .PARAMETER Force
+     Used to force the execution of a command.

```

Comment:
I think you need to be more specific than that. I would change it to:

`It will force the image generation when $RunSysprep is $False or the selected $SwitchName is not an external one. Use this parameter with caution because it can easily generate unstable images.`